doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,138 @@
+.. _gh_labels:
+
+Labeling issues and pull requests in GitHub
+###########################################
+
+Lots of GitHub issues and pull requests are being created daily.
+To help us manage them we have “labels”:
```

Comment:
nit: this colon (":") seems like it should be removed

Reply:
would a semicolon seem better?

Reply:
I don't think so. A semicolon would mean "to classify them by area, type, [...] attention" is an independent clause, which it is not (https://www.proofreadnow.com/blog/bid/71799/The-Semicolon-and-Independent-Clauses).

I don't think any punctuation marks are needed (or appropriate)

Reply:
Thanks, done